Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/ActiveEvent.cs

```diff
@@ -61,6 +61,12 @@ public GameObject AudioEmitter
             private set;
         }
 
+        public string MessageToSend
```

Comment:
What about using a proper unit event ? (see OnGazeEnterEvent for ex.)
That sounds cleaner to me instead of using string based messages.
Just my 2c.